markdown/ch3.markdown

```diff
@@ -200,7 +202,7 @@ def overlay_grid(image, spacing=128):
     image_gridded[:, spacing:-1:spacing] = [0, 0, 255]
     return image_gridded
 
-plt.imshow(overlay_grid(astro, 128))
+plt.imshow(overlay_grid(astro, 128));
```

Comment:
Do we want to add a callout box about [semicolons](https://www.youtube.com/watch?v=M94ii6MVilw) (=P) the first time we use this?